src/java.base/share/classes/jdk/internal/classfile/constantpool/ConstantPool.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package jdk.internal.classfile.constantpool;
+
+import jdk.internal.classfile.BootstrapMethodEntry;
+import jdk.internal.classfile.ClassReader;
+import jdk.internal.classfile.impl.ClassReaderImpl;
+
+import static jdk.internal.classfile.Classfile.TAG_CLASS;
+import static jdk.internal.classfile.Classfile.TAG_CONSTANTDYNAMIC;
+import static jdk.internal.classfile.Classfile.TAG_DOUBLE;
+import static jdk.internal.classfile.Classfile.TAG_FIELDREF;
+import static jdk.internal.classfile.Classfile.TAG_FLOAT;
+import static jdk.internal.classfile.Classfile.TAG_INTEGER;
+import static jdk.internal.classfile.Classfile.TAG_INTERFACEMETHODREF;
+import static jdk.internal.classfile.Classfile.TAG_INVOKEDYNAMIC;
+import static jdk.internal.classfile.Classfile.TAG_LONG;
+import static jdk.internal.classfile.Classfile.TAG_METHODHANDLE;
+import static jdk.internal.classfile.Classfile.TAG_METHODREF;
+import static jdk.internal.classfile.Classfile.TAG_METHODTYPE;
+import static jdk.internal.classfile.Classfile.TAG_MODULE;
+import static jdk.internal.classfile.Classfile.TAG_NAMEANDTYPE;
+import static jdk.internal.classfile.Classfile.TAG_PACKAGE;
+import static jdk.internal.classfile.Classfile.TAG_STRING;
+import static jdk.internal.classfile.Classfile.TAG_UTF8;
+
```

Comment:
Unused imports.

Reply:
This and hopefully all other unused imports have been removed.
Thanks.